src/Language/SystemVerilog/Parser/Lex.x

```diff
@@ -523,7 +523,7 @@ lexicalError :: String -> Alex a
 lexicalError msg = do
     (pn, _, _, _) <- alexGetInput
     pos <- toTokPos pn
-    alexError $ msg ++ ", at " ++ show pos
+    alexError $ show pos ++ "error: " ++ msg
```

Comment:
Can we change this to use `": Lexical error: "` in the style of the parser? Then we can remove that message prefix on line 501.

Reply:
Changed to `:Lexical error :` but given limited Haskell exposure I am not entirely sure how to use the `alexError` in the context in line 501.